nebula2/common/ttypes.py

```diff
@@ -117,7 +117,7 @@ class ErrorCode:
   E_BALANCED = -2024
   E_NO_RUNNING_BALANCE_PLAN = -2025
   E_NO_VALID_HOST = -2026
-  E_CORRUPTTED_BALANCE_PLAN = -2027
+  E_CORRUPTED_BALANCE_PLAN = -2027
```

Comment:
Note that this appears to relate to a change in vesoft-inc/nebula#3297 which definitely hints that it's an API and thus the typo should probably be retained.